uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -50,31 +50,68 @@ Feature: Greengrass V2 LogManager
-                    "logsUploaderConfiguration": {
-                        "componentLogsConfigurationMap": {
-                            "UserComponentA": {
-                                "logFileRegex": "UserComponentA_\\w*.log",
-                                "logFileDirectoryPath":"${UserComponentALogDirectory}",
-                                "deleteLogFileAfterCloudUpload":"true"
-                            }
-                        },
-                        "systemLogsConfiguration": {
-                            "uploadToCloudWatch":"true",
-                            "minimumLogLevel":"INFO",
-                            "diskSpaceLimit":"25",
-                            "diskSpaceLimitUnit":"MB",
-                            "deleteLogFileAfterCloudUpload":"true"
+         """
+        {
+            "MERGE": {
+                "logsUploaderConfiguration": {
+                     "componentLogsConfigurationMap": {
+                        "UserComponentA": {
+                            "logFileRegex": "UserComponentA_\\w*.log",
+                            "logFileDirectoryPath": "${UserComponentALogDirectory}",
+                            "deleteLogFileAfterCloudUpload": "true"
                         }
                     },
-                    "periodicUploadIntervalSec": "10"
-                }
+                    "systemLogsConfiguration": {
+                        "uploadToCloudWatch": "true",
+                        "minimumLogLevel": "INFO",
+                        "diskSpaceLimit": "25",
+                        "diskSpaceLimitUnit": "MB",
+                        "deleteLogFileAfterCloudUpload": "true"
+                    }
+                },
+                "periodicUploadIntervalSec": "10"
             }
-            """
+        }
+        """
         And I deploy the Greengrass deployment configuration
         Then the Greengrass deployment is COMPLETED on the device after 5 minutes
         Then I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
         And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
-        And I verify the rotated files are deleted and that the active log file is present for component UserComponentA
+        And I verify the rotated files are deleted except for the active log file for component UserComponentA
+
+    @R1    @functional @M2 @B1 @stable
+    Scenario: LogManager-1-T3: As a customer I can configure the logs uploader to delete log oldest log files inorder to keep the disk space limit configured by the customer
+        Given I create a Greengrass deployment with components
+            | aws.greengrass.Cli        | LATEST |
+            | aws.greengrass.LogManager | LATEST |
+        And I deploy the Greengrass deployment configuration
+        Then the Greengrass deployment is COMPLETED on the device after 4 minutes
```

Comment:
There are bunch of unnecessary steps here, you could remove line 87, 88, 89. See what we are doing for t1